dist/s_void

```diff
@@ -31,8 +31,8 @@ file_parse()
 func_ok()
 {
     sed \
+        -e '/int LLVMFuzzerTestOneInput$/d' \
         -e '/int __bm_stat$/d' \
-        -e '/int __bmd_free$/d' \
         -e '/int __bmd_stat$/d' \
```

Comment:
According to git blame this was added for disagg as well, shouldn we remove it?

https://github.com/wiredtiger/wiredtiger/blame/cd42e06513c9c82aa29e5a77ac176e247b77b3b5/dist/s_void#L36

Same for `__layered_table_manager_thread_run` below (https://github.com/wiredtiger/wiredtiger/blame/cd42e06513c9c82aa29e5a77ac176e247b77b3b5/dist/s_void#L55C30-L55C30)

Reply:
Unfortunately we still need these two -- not everything that got added during the big merge is a false positive.